marketing_security_group/security/mail_mass_mailing_security.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+<record id="mass_mailing_marketing_user_access" model="ir.rule">
+    <field name="name">Allows marketing user to delete only his not sent mail.mass_mailing objects</field>
```

Comment:
Actual name is better. marketing user != marketing manager, mail.mass_mailin != mails.

If you still want it shorter: *Marketing user - Delete own unsent mass mailings*